### DISCUSSION TITLE
How to move view center location position on Android?

### DISCUSSION THREAD

**Discussion by sergem155** [mapsforge/vtm#1140] [Q&A] [answered]
The viewport center is always 50% height and width.
I see no suitable methods in Viewport class :(
(I saw a similar question with an answer, but it looks related to another platform)

**Comment by devemux86** [comment, marked as answer]
It is [here](https://github.com/mapsforge/vtm/blob/master/vtm/src/org/oscim/map/ViewController.java#L117-L121):
```java
mapView.map().viewport().setMapViewCenterY(0.5);
```

**Comment by sergem155** [comment]
It works!

For some reason, in Kotlin the sequence of:

```
import org.oscim.map.Map
import org.oscim.map.Viewport
...
val map: Map = mapView.map()
val viewport: Viewport = mMap.viewport()
viewport.mapViewCenterY = 0.2F
```

gives the error about missing property mapViewCenterY (and associated setter and getter)
it also does not recognize tilt and a number of other properties, while recognizing maxTilt and minTilt

`mapView.map().viewport().mapViewCenterY = 0.5F`

indeed works perfectly

thank you!
